src/features/collaborator-avatars/CollaboratorAvatars.js

```diff
@@ -55,20 +56,29 @@ class CollaboratorAvatars extends Component<Props> {
             : `+${collaborators.length - maxDisplayedUserAvatars}`;
     }
 
+    handleActivate = (event: SyntheticKeyboardEvent<HTMLAnchorElement> | SyntheticMouseEvent<HTMLAnchorElement>) => {
+        const { onClick } = this.props;
+        // bail if it's a keyboard event but not the enter key
+        if (event.key && event.key !== KEYS.enter) {
+            return;
+        }
+        onClick();
+    };
+
     render() {
-        const { collaborators, maxDisplayedUserAvatars, containerAttributes, onClick } = this.props;
+        const { collaborators, maxDisplayedUserAvatars, containerAttributes } = this.props;
 
         return (
-            // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
             <a
```

Comment:
Should this just be a button then? PlainButton?

Reply:
I agree.  Then you can also remove the role="button"

Reply:
Yep, this is better. Done.